doc/source/skips/0-skip-process.rst

```diff
@@ -0,0 +1,300 @@
+.. _skip0:
+
+============================
+SKIP 0 — Purpose and Process
+============================
+
+:Author: Jarrod Millman <millman@berkeley.edu>
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Status: Active
+:Type: Process
+:Created: 2019-07-30
+
+
+What is a SKIP?
+--------------
+
+SKIP stands for SciKit-Image Proposal. A SKIP is a design document providing
+information to the NumPy community, or describing a new feature for
+scikit-image or its processes or environment. The SKIP should provide a
+concise technical specification of the feature and a rationale for the feature.
```

Comment:
This sentence only applies when it is a feature SKIP.

Reply:
Updated.